### PR TITLE
Update harvard-de-montfort-university.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/67483/de-montfort-university-harvard-style-update

Not ready, I think. Requestor is waiting to hear back from their library.

### CODE REVIEW

File: harvard-de-montfort-university.csl [hunk 1]
```diff
@@ -134,88 +127,158 @@
...
+              <date date-parts="year" form="text" variable="issued" prefix="(" suffix=")"/>
+              <group delimiter=", ">
+                <text variable="number" prefix="Episode "/>
+                <text variable="container-title" suffix="."/>
+              </group>
+              <text variable="medium" text-case="capitalize-first" prefix="[" suffix="]"/>
+              <text variable="publisher"/>
+            </group>
+            <group delimiter=", ">
+              <date form="text" date-parts="month-day" variable="issued">
```
Comment (adam3smith) [inline harvard-de-montfort-university.csl]: this is why the validation fails -- you can either do the date-parts here or below, but not both.

**Comment by rmzelle** [comment]
@POBrien333, ready?

**Comment by POBrien333** [comment]
Yes. :)

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**